app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainFragment.kt

```diff
@@ -51,7 +51,7 @@ class MainFragment : Fragment() {
     }
 
     lateinit var mFusedLocationProviderClient: FusedLocationProviderClient
-    val locationRequestId = 100
+    val LOCATION_REQUEST_ID = 100
```

Comment:
Simply renaming this variable to better reflect coding practices when defining constant variables :)

https://medium.com/incwell-bootcamp/declaring-variables-in-kotlin-64d6261144ae

Reply:
I think I fix this but may change back due to merging that we did going to master. Still a great catch.